swarm/tests/connection_close.rs

```diff
@@ -103,7 +103,7 @@ impl ConnectionHandler for HandlerWithState {
     type InboundOpenInfo = ();
     type OutboundOpenInfo = ();
 
-    fn listen_protocol(&self) -> SubstreamProtocol<Self::InboundProtocol, Self::InboundOpenInfo> {
+    fn listen_protocol(&self) -> SubstreamProtocol<Self::InboundProtocol, ()> {
```

Comment:
I wonder: Wdyt of making the generic default to `()`, like `SubstreamProtocol<_, TInfo = ()>`? That way it would already disappear from most APIs.